File: scripts/remove-duplicates.rb
#!/usr/bin/env ruby

ARGV.each do |a|
  begin
    puts "Editing file: #{a}"

    lines = File.readlines(a).uniq

    File.open(a, 'w+') do |f|
      f.puts(lines)
    end
  rescue StandardError
    puts "| Could not open file: #{a}"
    next
  end
end

File: scripts/remove-duplicates.py
#!/usr/bin/env python3

import sys

files = sys.argv
files.pop(0)

for file in files:
    print("Editing file: " + file)
    lines_seen = []

    for line in open(file, "r"):
        if line not in lines_seen:
            lines_seen.append(line)

    file = open(file, 'w')
    file.writelines(lines_seen)
    file.close()
